### PR TITLE
Resprites RPD and RCD

### PR DESCRIPTION
**Pull request by Michiyamenotehifunana**
dreamseeker_B4bHZ6WLMY dreamseeker_GNvV6gpjhV

This PR will resprite the RCD and RPD, and also actually give the RPD inhands. Both sprites are my own, and have been used on various other servers before. Fun fact: the RCD sprite that this PR is replacing also happens to be one of my earlier sprites.

:cl:  Toriate
rscadd: RPD now has inhands
imageadd: RCD and RPD have been resprited
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
cool

**Comment by Marmio64** [comment]
i really like them but they dont really fit in with the art style

**Comment by Nickvr628** [comment]
They are different but I like them. Having the blue "materials meter" on them is a good addition.

**Comment by Hopekz** [comment]
What's this part supposed to represent?

The area before seemed like a nice angled grip now it seems hard to imagine this being welded with both hands and it still looks like a heavy tool.

image

**Comment by Featherfield** [comment]
Dont fit the art style at all

**Comment by Partheo** [comment]
Tori!!!!!!!!!!! So happy to see you.

Thank you for blessing us with your presence. What made you come here? (besides the absence of good RPD/RCD sprites)

(Also: join us in the spriter discord. we need you)

**Comment by TheBimmer231** [comment]
Ayyyyy Tori's alive! gorgeous sprites! but maybe... too good for yogstation?

File: code/game/objects/items/RPD.dm [hunk 1]
```diff
@@ -193,6 +193,8 @@ GLOBAL_LIST_INIT(fluid_duct_recipes, list(
 	name = "Rapid Pipe Dispenser (RPD)"
 	desc = "A device used to rapidly pipe things."
 	icon = 'icons/obj/tools.dmi'
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
```
Comment (Partheo) [inline code/game/objects/items/RPD.dm:197]: My man added inhands, 10/10

**Review by Partheo — APPROVED** [review]

**Review by nesisknesis — APPROVED** [review]

**Pull request merged**